packages/commonwealth/client/scripts/views/modals/canvas_verify_data_modal.tsx

```diff
@@ -0,0 +1,115 @@
+/* @jsx m */
+
+import m from 'mithril';
+import $ from 'jquery';
+import {
+  serializeActionPayload,
+  serializeSessionPayload,
+} from '@canvas-js/interfaces';
+import type { Action, Session } from '@canvas-js/interfaces';
+
+import app from 'state';
+import { CWButton } from '../components/component_kit/cw_button';
+import { verify } from '../../helpers/canvas';
+
+import 'modals/canvas_verify_data_modal.scss';
+
+const CanvasVerifyDataModal = {
```

Comment:
Do we have screenshots for how this looks? Should be audited by product (cc @zakhap).

Reply:
Will include these in the PR to turn the modal on.